echopype/preprocess/api.py

```diff
@@ -62,16 +62,20 @@ def compute_MVBS(ds_Sv, range_meter_bin=20, ping_time_bin="20S"):
     A dataset containing bin-averaged Sv
     """
 
-    if not ds_Sv.groupby("frequency").apply(_check_range_uniqueness).all():
+    if not ds_Sv.groupby("channel").apply(_check_range_uniqueness).all():
         raise ValueError(
-            "echo_range variable changes across pings in at least one of the frequency channel."
+            "echo_range variable changes across pings in at least one of the frequency channels."
         )
 
+    # get indices of sorted frequency_nominal values. This is necessary
+    # because the frequency_nominal values are not always in ascending order.
+    sorted_freq_ind = np.argsort(ds_Sv.frequency_nominal)
+
     def _freq_MVBS(ds, rint, pbin):
         sv = 10 ** (ds["Sv"] / 10)  # average should be done in linear domain
         sv.coords["range_meter"] = (
             ["range_sample"],
-            ds_Sv["echo_range"].isel(frequency=0, ping_time=0).data,
+            ds_Sv["echo_range"].isel(channel=sorted_freq_ind[0], ping_time=0).data,
         )
```

Comment:
This needs some clean-up, but let's make it a different PR after v0.6.0. I can see the operations you implemented correctly replicated the original behavior, so let's keep this here for now.

I've added #662 to keep track of this.